core/src/com/unciv/logic/civilization/Civilization.kt

```diff
@@ -191,6 +197,7 @@ class Civilization : IsPartOfGameInfoSerialization {
     var cities = listOf<City>()
     var citiesCreated = 0
     var exploredTiles = HashSet<Vector2>()
+    var exploredRegion = HashMap<String,Vector2>()
```

Comment:
Since there are only 2 vectors , this should not be a dictionary, treat them by name.
Also, call them 'topright' and 'bottomleft', 'start' and 'end' is very ambiguous

Reply:
Also, *this can be calculated from tilemap data*, so we don't need to persist this at all! All these can be transient!
(Since there are several vectors you can store them all in an object, like 'class exploredRegion')

Reply:
> Also, _this can be calculated from tilemap data_, so we don't need to persist this at all! All these can be transient!

tested it, tiles become explored in a random order, sometimes messing up the explored region.